lib/src/main/java/uk/gov/di/ipv/core/library/helpers/RequestHelper.java

```diff
@@ -95,4 +101,16 @@ public static String getIpvSessionId(Map<String, String> headers)
         LogHelper.attachIpvSessionIdToLogs(ipvSessionId);
         return ipvSessionId;
     }
+
+    public static String getClientSourceIp(Map<String, String> headers)
+            throws HttpResponseExceptionWithErrorBody {
+        String ipvSessionId = RequestHelper.getHeaderByKey(headers, CLIENT_SOURCE_IP_HEADER);
```

Comment:
Does this need a rename -> `clientSourceIp` ?

Reply:
Done